plasma_framework/contracts/src/exits/payment/controllers/PaymentChallengeStandardExit.sol

```diff
@@ -0,0 +1,87 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "../PaymentExitDataModel.sol";
+import "../routers/PaymentStandardExitRouterArgs.sol";
+import "../spendingConditions/IPaymentSpendingCondition.sol";
+import "../spendingConditions/PaymentSpendingConditionRegistry.sol";
+import "../../../vaults/EthVault.sol";
+import "../../../vaults/Erc20Vault.sol";
+import "../../../framework/PlasmaFramework.sol";
+
+library PaymentChallengeStandardExit {
+    struct Controller {
+        PlasmaFramework framework;
+        PaymentSpendingConditionRegistry spendingConditionRegistry;
+        uint256 standardExitBond;
+    }
+
+    event ExitChallenged(
+        uint256 indexed utxoPos
+    );
+
+    /**
+     * @dev data to be passed around helper functions
+     */
+    struct ChallengeStandardExitData {
+        Controller controller;
+        PaymentStandardExitRouterArgs.ChallengeStandardExitArgs args;
+        PaymentExitDataModel.StandardExit exitData;
+    }
+
+    function run(
+        Controller memory self,
```

Comment:
Sometimes we append a dash '_' to argument name and sometimes we don't. Is there a rule? I prefer not to use dashes at all.

Reply:
nah...I don't think there's a rule on this. That's why this issue: https://github.com/omisego/plasma-contracts/issues/191.

I was just following previous convention that uses `_` but in general I agree with you in that I don't really see value of those underscore.

The only time I think it has value is when we have private variable but want some getter and not using the default getter logic. If we don't underscore those var, we cannot provide getter with the style `varname()` as it is occupied by compiler (even you make it private :(